### ISSUE TITLE
Address remaining feedback of "Support RC2" PR

### ISSUE THREAD

**Issue by julianbrost** [Icinga/docker-icingadb#29]
#26 was merged to get working images, but there is some remaining feedback to be addressed: https://github.com/Icinga/docker-icingadb/pull/26#pullrequestreview-678545501

**Comment by Al2Klimov** [comment]
Not quite sure what’s not addressed, yet. Please add a list of short bullet points (preferably w/ checkboxes).

**Comment by julianbrost** [comment]
* [ ] However, when running the container without any environment variables set, it just segfaults. This could be handled in a more graceful way.
  ```
  $ docker run --rm -it icinga/icingadb:master
  2021-06-08T13:47:17.710Z	DEBUG	entrypoint/main.go:40	translating env vars to YaML config
  2021-06-08T13:47:17.710Z	DEBUG	entrypoint/main.go:85	checking SQL database
  panic: runtime error: invalid memory address or nil pointer dereference
  [signal SIGSEGV: segmentation violation code=0x1 addr=0x28 pc=0x6bfdc2]

  goroutine 1 [running]:
  github.com/icinga/icingadb/pkg/config.(*Database).Open(0x0, 0xc00000e178, 0xc0001a8800, 0x0, 0x0)
	/go/pkg/mod/github.com/icinga/icingadb@v1.0.0-rc1.0.20210528122710-9fdcc839798a/pkg/config/database.go:37 +0x82
  main.initDb(0x0, 0x0)
	/entrypoint/main.go:92 +0x16a
  main.runDaemon(0x7504f0, 0x0)
	/entrypoint/main.go:75 +0x165
  main.main()
	/entrypoint/main.go:36 +0x37
  ```
* [ ] I haven't seen this capitalization of YAML before. https://yaml.org/ also uses all uppercase. (https://github.com/Icinga/docker-icingadb/pull/26#discussion_r647461153)

**Comment by lippserd** [comment]
This one can be closed because of #34 and Icinga/icingadb#303, right?

**Comment by julianbrost** [comment]
Indeed
